### PR TITLE
Adds epic and also cool paramedics [lazy tg port]

### PR DESCRIPTION
**Pull request by Detective-Google**
## About The Pull Request

ports paramedics from /tg/
## Why It's Good For The Game

paramedics are neat.
or per the /tg/ pr:

Right now, doctors can either hang around medbay (boring) or go out and look for injured people, which keeps them from treating patients that show up at medbay. paramedics are the counterpart that roams the station, bringing in injured patients.
I think that this addition will bring MDs more patients and provide a job for people who want to help other players without being expected to do surgery.

## Changelog
:cl:
add:Paramedics
/:cl:

### CODE REVIEW

**Comment by Detective-Google** [comment]
current issue with this pr; 
image
not reallly sure how to fix this except maybe shunting Sci+Sec+Silicons to the right panel or just Sec+Silicons.

**Comment by Detective-Google** [comment]
fixed it by putting silicons on the left in the extra 2 empty slots.

**Comment by YakumoChen** [comment]
...this just seems like "MD but better" to be honest. They lack surgery access, but in return they get better access then every non-head job in the game. Hell, most heads of staff don't even get EVA access, which this does.

**Comment by Detective-Google** [comment]
> ...this just seems like "MD but better" to be honest. They lack surgery access, but in return they get better access then every non-head job in the game. Hell, most heads of staff don't even get EVA access, which this does.

1. I see no problem with it literally being MD but better.
2. tg has it as such and they actually care about balance.
I could point out that jobs like engineers and atmos techs have really good access as do geneticists.

**Comment by CameronWoof** [comment]
Cargo? Hydroponics? What use do they have of accesses like these?

File: code/modules/jobs/job_types/paramedic.dm [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+	minimal_access = list(ACCESS_MEDICAL, ACCESS_MORGUE, ACCESS_CLONING, ACCESS_MINERAL_STOREROOM, ACCESS_MAINT_TUNNELS, ACCESS_EVA, ACCESS_ENGINE, ACCESS_CARGO, ACCESS_HYDROPONICS, ACCESS_RESEARCH)
+
+	display_order = JOB_DISPLAY_ORDER_PARAMEDIC
+
+/datum/outfit/job/paramedic
+	name = "Paramedic"
+	jobtype = /datum/job/paramedic
+
+	ears = /obj/item/radio/headset/headset_med
+	gloves = /obj/item/clothing/gloves/color/latex/nitrile
```
Comment (Trilbyspaceclone) [inline code/modules/jobs/job_types/paramedic.dm:24]: Give them normal latex as nitrile is DNA-less

File: code/modules/jobs/job_types/paramedic.dm [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+	department_flag = MEDSCI
+	faction = "Station"
+	total_positions = 3
+	spawn_positions = 2
+	supervisors = "the chief medical officer"
+	selection_color = "#74b5e0"
+
+	outfit = /datum/outfit/job/paramedic
+
+	access = list(ACCESS_MEDICAL, ACCESS_MORGUE, ACCESS_SURGERY, ACCESS_GENETICS, ACCESS_CLONING, ACCESS_MINERAL_STOREROOM, ACCESS_MAINT_TUNNELS, ACCESS_EVA, ACCESS_ENGINE, ACCESS_CARGO, ACCESS_HYDROPONICS, ACCESS_RESEARCH)
```
Comment (Trilbyspaceclone) [inline code/modules/jobs/job_types/paramedic.dm]: Cargo/Hydro/Engi/Sci are not needed
EVA is reasonable but maybe medical like all departments should get a hardsuit or two?

File: _maps/map_files/BoxStation/BoxStation.dmm [hunk 3]
```diff
@@ -57002,17 +56686,6 @@
...
-/obj/machinery/atmospherics/pipe/simple/supply/hidden{
-	dir = 4
-	},
-/turf/open/floor/plasteel,
-/area/crew_quarters/fitness/pool)
-"rje" = (
-/obj/machinery/atmospherics/pipe/simple/supply/hidden,
-/obj/structure/bed,
-/turf/open/floor/plasteel,
-/area/crew_quarters/fitness/pool)
```
Comment (Trilbyspaceclone) [inline _maps/map_files/BoxStation/BoxStation.dmm]: Why are you reverting the pool?

File: code/modules/jobs/job_types/paramedic.dm [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+	minimal_access = list(ACCESS_MEDICAL, ACCESS_MORGUE, ACCESS_CLONING, ACCESS_MINERAL_STOREROOM, ACCESS_MAINT_TUNNELS, ACCESS_EVA, ACCESS_ENGINE, ACCESS_CARGO, ACCESS_HYDROPONICS, ACCESS_RESEARCH)
+
+	display_order = JOB_DISPLAY_ORDER_PARAMEDIC
+
+/datum/outfit/job/paramedic
```
Comment (Trilbyspaceclone) [inline code/modules/jobs/job_types/paramedic.dm:19]: Is their even any way to replace their outfits if they lose/need more gear?

**Review by Trilbyspaceclone — CHANGES_REQUESTED** [review]

**Comment by necromanceranne** [comment]
Works a little better on tg due to not having ready access to surgery tools for tend wounds and stasis beds preventing any shmuck from performing medical miracles in the middle of no where. Additionally, upgrades to paramedic gear cost money.

Still a fun role.

File: code/modules/jobs/job_types/paramedic.dm (hunk 2 above)
Comment (YakumoChen) [reply code/modules/jobs/job_types/paramedic.dm]: EVA isn't reasonable, the only people with default access to EVA are the captain and two heads of staff. Just give them their own softsuit (they're doctors, they don't need a hardsuit) if you really want them to be EVA-ready at shift start.

**Comment by ghost** [comment]
sounds fun
if doctors didn't fight for patients outside of medbay on a slow shift

**Review by silicons — APPROVED** [review]

**Pull request merged**